darwin/importer/formats/coco.py

```diff
@@ -0,0 +1,58 @@
+import json
+from pathlib import Path
+from typing import List, Optional
+
+import darwin.datatypes as dt
+
+
+def parse_file(path: Path) -> Optional[List[dt.AnnotationFile]]:
+    if path.suffix != ".json":
+        return
+
+    with path.open() as f:
+        data = json.load(f)
+        return list(parse_json(path, data))
+
+
+def parse_json(path, data):
+    annotations = data["annotations"]
+    image_lookup_table = {image["id"]: image for image in data["images"]}
+    category_lookup_table = {category["id"]: category for category in data["categories"]}
+    image_annotations = {}
+
+    for annotation in annotations:
+        image_id = annotation["image_id"]
+        annotation["category_id"]
+        annotation["segmentation"]
+        if image_id not in image_annotations:
+            image_annotations[image_id] = []
+        image_annotations[image_id].append(parse_annotation(annotation, category_lookup_table))
+
+    for image_id in image_annotations.keys():
+        image = image_lookup_table[image_id]
+        annotations = list(filter(None, image_annotations[image_id]))
+        annotation_classes = set([annotation.annotation_class for annotation in annotations])
+        yield dt.AnnotationFile(path, image["file_name"], annotation_classes, annotations)
+
+
+def parse_annotation(annotation, category_lookup_table):
+    category = category_lookup_table[annotation["category_id"]]
+    segmentation = annotation["segmentation"]
+    iscrowd = annotation["iscrowd"] == 1
```

Comment:
I've seen some recent datasets (eg. LVIS) that use COCO's format that don't include `iscrowd` anymore, so I'd replace this line with:

```
iscrowd = annotation.get("iscrowd") == 1
```